src/playercard/plot.py

```
@@ -2,29 +2,41 @@
+        "line-color",
+    )
+    TEXT_COLOR = THEME.find_stylesheet_attribute(
+        "RatingPlotCrosshairs::custom",
+        "text-color",
+    )
+    BACKGROUND_COLOR = THEME.find_stylesheet_attribute(
+        "RatingPlotCrosshairs::custom",
+        "background-color",
+    )
+
     def __init__(self, plotwidget: pg.PlotWidget, series: LineSeries) -> None:
```

Ruff (0.14.13)

Undefined name `LineSeries`

(F821)